Dockerfile

```diff
@@ -30,8 +30,9 @@ RUN chown -R $USER_ID_GROUP /vendor
 USER $USER_ID_GROUP
 
 # Upgrade RubyGems and install required Bundler version
-RUN gem update --system && \
-      gem install bundler:$BUNDLER_VERSION
+# https://github.com/rubygems/rubygems/issues/2534#issuecomment-448843746
+RUN gem update --system --conservative || (gem i "rubygems-update:~>2.7" --no-document && update_rubygems) && \
```

Comment:
does the 2.7 need to be updated too now that we added some newer versions of Ruby?

Reply:
This change fixed the behaviour for all `jruby` images and `ruby<=2.5` images. and the updated matrix list is not in that range. Hence, I think this is still needed. But to be really honest, I only copied the command from the corresponding [comment](https://github.com/rubygems/rubygems/issues/2534#issuecomment-448843746), so I'm not sure how and if this command can be altered. 

Please let me know if you have other ideas and I will happily apply them. But all tests are green right now.

Reply:
Ok, thanks for the explanation. If the tests are green, I'm ok with this.